slides/07-Platform-Hub/1070-ContinuingProjectsAndKubeTokens.md

```diff
@@ -0,0 +1,14 @@
+## Continuing Projects and Kube tokens
+
+ - We've now allocated developer-induction kubernetes tokens
+ - These can be seen in Connected Identities. To see it expand Show Tokens and click on the eye icon
+ - You'll need to update the kube config by running the following commands:
+
+<span style="line-height: .4em; color: blue; font-size: 0.5em; display: block;">kubectl config set-cluster acp-dev --certificate-authority=&lt;acp-dev-cert&gt; --server=&lt;acp-dev-cluster-ip&gt;</span>
```

Comment:
any chance of putting these in a code block on their own page like how we do on https://raw.githubusercontent.com/UKHomeOffice/developer-induction/master/slides/08-Continuous-Integration/1020-example-drone-yaml.md